src/ExpressReceiver.ts

```diff
@@ -52,9 +72,49 @@ export default class ExpressReceiver implements Receiver {
 
     this.processBeforeResponse = processBeforeResponse;
     this.logger = logger;
-    const endpointList: string[] = typeof endpoints === 'string' ? [endpoints] : Object.values(endpoints);
+    const endpointList = typeof endpoints === 'string' ? [endpoints] : Object.values(endpoints);
+    this.router = Router();
     for (const endpoint of endpointList) {
-      this.app.post(endpoint, ...expressMiddleware);
+      this.router.post(endpoint, ...expressMiddleware);
+    }
+    this.app.use(this.router);
+    this.oauthAuthorize = undefined;
+
+    let oauthInstaller: OAuthInterface | undefined = undefined;
```

Comment:
let's make this a public property of `ExpressReceiver`. this will give users the ability to call `generateInstallUrl()` to make URLs for subsequent authorizations. the property name could be `installer`.